frontend/src/service/apiService.ts

```diff
@@ -1,23 +1,56 @@
-import {game} from "./model";
+import {game, LoginData, LoginResponse, MyUserCreationData} from "./model";
 import axios, {AxiosResponse} from "axios";
 
 export function createAdminGame(game: game){
-    return axios.post("/api/game/admin", game)
+    return axios.post("/api/game/admin", game,{
+        headers:{
+            Authorization: "Bearer " + localStorage.getItem("jwt")
+        }
+    })
 }
```

Comment:
Wenn du hier am Anfang der Datei einmal die configuration in einer Variable speicherst, musst du die danach nicht immer pasten. und fallst du mal wa an der config verändern willst, ist dann auch viel einfacher: 

```Javascript
let requestConfig = {
    headers: {
        Authorization: `Bearer ${localStorage.getItem("jwt-alexandra")}`
    }
}
 export function createAdminGame(game: game) {
    return axios.post("api/game/admin", game, requestConfig) 
}
```
Das gilt dann natürlich auch für alle anderen funktionen in dieser Datei, außer die login und register funtionen, die haben ja kein token.